### DISCUSSION TITLE
cache clear command issue in prod env

### DISCUSSION THREAD

**Discussion by AltState** [symfony/symfony#49465] [Q&A]
Hello,

we have a project under _Symfony 6.2, PHP 8.2_ hosted on _Windows Server 2016 with Apache 2.4_
Every time _cache:clear command_ is launched, a lot of errors like this appears in the _prod / php-errors files_ and the server completely freezes (requests timeout). 
It can takes a dozen of minutes before the command finally ends.

Here is a snippet from logs files :
`C:\project_dir\vendor\doctrine\dbal\src\Driver\PDO\Statement.php on line 121
[21-Feb-2023 10:49:53 Europe/Paris] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in C:\project_dir\vendor\doctrine\dbal\src\Driver\PDO\Statement.php on line 121
[21-Feb-2023 10:49:54 Europe/Paris] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in C:\project_dir\vendor\doctrine\dbal\src\Driver\PDO\Statement.php on line 121
[21-Feb-2023 10:49:54 Europe/Paris] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in C:\project_dir\vendor\doctrine\dbal\src\Driver\PDO\Statement.php on line 121`
`........`
`[21-Feb-2023 10:54:03 Europe/Paris] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in C:\project_dir\vendor\symfony\cache\Traits\FilesystemCommonTrait.php on line 107
[21-Feb-2023 10:54:03 Europe/Paris] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in C:\project_dir\vendor\symfony\cache\Traits\FilesystemCommonTrait.php on line 107
[21-Feb-2023 10:54:03 Europe/Paris] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in C:\project_dir\vendor\symfony\cache\Traits\FilesystemCommonTrait.php on line 161
[21-Feb-2023 10:54:03 Europe/Paris] PHP Fatal error:  Maximum execution time of 30 seconds exceeded in C:\project_dir\vendor\symfony\cache\Traits\FilesystemCommonTrait.php on line 161`

Our composer.json : 
`{
    "type": "project",
    "license": "proprietary",
    "require": {
        "php": "^8.2",
        "ext-ctype": "*",
        "ext-iconv": "*",
        "ext-intl": "*",
        "ext-json": "*",
        "ext-ftp": "*",
        "beberlei/doctrineextensions": "^1.3",
        "composer/package-versions-deprecated": "1.11.99.1",
        "doctrine/doctrine-bundle": "^2.8",
        "doctrine/orm": "^2.14",
        "friendsofsymfony/rest-bundle": "^3.4",
        "gesdinet/jwt-refresh-token-bundle": "^1.1",
        "lexik/jwt-authentication-bundle": "^2.16",
        "mustache/mustache": "^2.14",
        "nelmio/cors-bundle": "^2.2",
        "osimatic/helpers-php": "dev-master",
        "sensio/framework-extra-bundle": "^6.2",
        "symfony/asset": "~6.2",
        "symfony/console": "~6.2",
        "symfony/dotenv": "~6.2",
        "symfony/expression-language": "~6.2",
        "symfony/flex": "^2",
        "symfony/framework-bundle": "~6.2",
        "symfony/mime": "~6.2",
        "symfony/monolog-bundle": "^3.8",
        "symfony/translation": "~6.2",
        "symfony/twig-bundle": "~6.2",
        "symfony/validator": "~6.2",
        "symfony/yaml": "~6.2",
        "twig/extra-bundle": "^3.5",
        "twig/twig": "^3.5"
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": {
            "*": "dist"
        },
        "sort-packages": true,
        "allow-plugins": {
            "symfony/flex": true
        }
    },
    "autoload": {
        "psr-4": {
            "App\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "App\\Tests\\": "tests/"
        }
    },
    "replace": {
        "paragonie/random_compat": "2.*",
        "symfony/polyfill-ctype": "*",
        "symfony/polyfill-iconv": "*",
        "symfony/polyfill-php72": "*",
        "symfony/polyfill-php71": "*",
        "symfony/polyfill-php70": "*",
        "symfony/polyfill-php56": "*"
    },
    "scripts": {
        "auto-scripts": {
            "cache:clear": "symfony-cmd",
            "assets:install %PUBLIC_DIR%": "symfony-cmd"
        },
        "post-install-cmd": [
            "@auto-scripts"
        ],
        "post-update-cmd": [
            "@auto-scripts"
        ]
    },
    "conflict": {
        "symfony/symfony": "*"
    },
    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "~6.2"
        }
    }
}`

Thanks for your help.
Kind regards

**Comment by uncaught** [comment]
You should stop traffic during cache rebuild. As far as I know there is no locking mechanism, so every request running into an empty cache will trigger the rebuild simultaneously.
